src/queryBookmark.ts

```diff
@@ -3,7 +3,7 @@ type inlineObject = any;
 
 // is inlineObject a bookmark?
 export const isBookmark: (inlineObject: inlineObject) => inlineObject | boolean = (inlineObject) => {
-	if (inlineObject.bookmarkType !== undefined) {
+	if (inlineObject && inlineObject.bookmarkType !== undefined) {
```

Comment:
should not need to do this. because `inlineObject` is not a optional argument according to the type definition. if your test was a ts file, you would not be able to do `isBookmark(undefined)`.

Reply:
@p-mag @AdamTombleson passing inlineObject as undefined doesn't return false. I have changed some syntax to make it work. Please review changes.

Reply:
I have changed test file from js to ts before making inlineObject optional. It doesn't work. @AdamTombleson